.gitignore

```diff
@@ -12,3 +12,5 @@ node_modules/
 /test/sandbox
 /test/.pkgcache/
 dist
+package-lock.json
```

Comment:
We don't use package lock. What to do instead: add `.npmrc` file disabling this feature. You can copy this file e.g. from the `loopback` module, see https://github.com/strongloop/loopback/blob/master/.npmrc